portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +32,18 @@ function topFunction() {
   document.documentElement.scrollTop = 0;
 }
 
+var getData;
+// External Citation:
+// https://www.quora.com/How-can-I-make-a-comment-box-in-HTML
+(getData = function getData(){
+    fetch('/data').then(response => response.json()).then((data) => {
+        console.log(data[0]);
+        var displayComments = document.getElementById('display-comments');
+        var comment;
+        for(comment in data){
```

Comment:
`comment` here is a bit misleading for a variable name – I believe `comment` in this case is an index because you're using a `for in` loop.

You can either do `for (comment of data)`, in which case you won't have to do `data[comment]` to access the actual element of the array, or you'd like to actually use the index, then it'd be clearer to the reader to name this variable `index` or `commentIdx` or something to that effect.

https://bitsofco.de/for-in-vs-for-of/ for more explanation.

Reply:
got it.